### PR TITLE
Update data-import-product-images.md

### PR DESCRIPTION
**Pull request by hickskenh**
## Purpose of this pull request

This pull request (PR) updates the paragraph about importing multiple images to a single SKU.

## Affected documentation pages

- https://docs.magento.com/m2/ee/user_guide/system/data-import-product-images.html?itm_source=merchdocs_ee&itm_medium=quick_search&itm_campaign=federated_search&itm_term=

## Affected Magento editions

- [ X] Open Source
- [X] Commerce
- [X] B2B

## Additional information

The paragraph about importing multiple images to a single SKU is incorrect, as it's not the supported method anymore. The currently correct way is described here: https://magento.stackexchange.com/questions/212785/how-to-import-multiple-images-for-a-single-sku

### CODE REVIEW

File: src/system/data-import-product-images.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Before you try to import a large amount of images, review the steps in the impor
     ![Data import images file directory]({% link images/images/data-import-images-file-to-import.png %}){: .zoom}
     _Images File Directory_
 
-    If importing multiple images for a single `sku`, insert a blank row below the `sku` and enter the additional image file names in the appropriate columns. The additional rows are understood to belong to the parent `sku`.
+    If importing multiple images for a single `sku`, insert them in a column named `additional_images`, separated by commas. Example: `image02.jpg,image03.jpg`.
```
Comment (hguthrie) [inline src/system/data-import-product-images.md]: Is "them" the image file name? It is best to say something like:

```suggestion
    If importing multiple images for a single `sku`, insert the images in a column named `additional_images`, separated by commas. Example: `image02.jpg,image03.jpg`.
```

Comment (hickskenh) [reply src/system/data-import-product-images.md]: accepted changes.  added per the column (add the column if not already added).

**Comment by dshevtsov** [comment]
FYI
To copy a clean URL to a topic avoiding parameters such as `?itm_source=merchdocs_ee&itm_medium=quick_search&itm_campaign=federated_search&itm_term=`, you can use **Copy link address** from the dropdown menu **at the topic's title**.
copy-clean-url

File: src/system/data-import-product-images.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ Before you try to import a large amount of images, review the steps in the impor
     ![Data import images file directory]({% link images/images/data-import-images-file-to-import.png %}){: .zoom}
     _Images File Directory_
 
-    If importing multiple images for a single `sku`, insert a blank row below the `sku` and enter the additional image file names in the appropriate columns. The additional rows are understood to belong to the parent `sku`.
+    If importing multiple images for a single `sku`, insert the images in a column named `additional_images` (add the column if not already added), separated by commas. Example: `image02.jpg,image03.jpg`.
```
Comment (hguthrie) [inline src/system/data-import-product-images.md]: I would remove the period after the example so users don't emulate. Unless of course it's required...your call.

```suggestion
    If importing multiple images for a single `sku`, insert the images in a column named `additional_images` (add the column if not already added), separated by commas. Example: `image02.jpg,image03.jpg`
```

**Comment by hickskenh** [comment]
Thanks Heather and Dima.  Heather, you're right, the example isn't a sentence and shouldn't have a .

**Comment by hickskenh** [comment]
running tests

**Pull request merged**